Examples/Python/python/acts/examples/reconstruction.py

```diff
@@ -663,6 +671,11 @@ def addSpacePointsMaking(
         geometrySelection=acts.examples.readJsonGeometryList(
             str(geoSelectionConfigFile)
         ),
+        stripGeometrySelection=(
```

Comment:
I didn't realize that we have two params for this. maybe something we should revisit at some point

Reply:
There seems to be a god reason for it. I.e. If no strips are required whole strip module pairs map configuration is skipped.